local_node.sh

```diff
@@ -34,108 +34,118 @@ set -e
-
-# If keys exist they should be deleted
-for KEY in "${KEYS[@]}"; do
-	evmosd keys add $KEY --keyring-backend $KEYRING --algo $KEYALGO --home "$HOMEDIR"
-done
-
-# Set moniker and chain-id for Evmos (Moniker can be anything, chain-id must be an integer)
-evmosd init $MONIKER -o --chain-id $CHAINID --home "$HOMEDIR"
-
-# Change parameter token denominations to aevmos
-cat "$GENESIS" | jq '.app_state["staking"]["params"]["bond_denom"]="aevmos"' >"$TMP_GENESIS" && mv "$TMP_GENESIS" "$GENESIS"
-cat "$GENESIS" | jq '.app_state["crisis"]["constant_fee"]["denom"]="aevmos"' >"$TMP_GENESIS" && mv "$TMP_GENESIS" "$GENESIS"
-cat "$GENESIS" | jq '.app_state["gov"]["deposit_params"]["min_deposit"][0]["denom"]="aevmos"' >"$TMP_GENESIS" && mv "$TMP_GENESIS" "$GENESIS"
-cat "$GENESIS" | jq '.app_state["evm"]["params"]["evm_denom"]="aevmos"' >"$TMP_GENESIS" && mv "$TMP_GENESIS" "$GENESIS"
-cat "$GENESIS" | jq '.app_state["inflation"]["params"]["mint_denom"]="aevmos"' >"$TMP_GENESIS" && mv "$TMP_GENESIS" "$GENESIS"
-
-# Set gas limit in genesis
-cat "$GENESIS" | jq '.consensus_params["block"]["max_gas"]="10000000"' >"$TMP_GENESIS" && mv "$TMP_GENESIS" "$GENESIS"
-
-# Set claims start time
-current_date=$(date -u +"%Y-%m-%dT%TZ")
-cat "$GENESIS" | jq -r --arg current_date "$current_date" '.app_state["claims"]["params"]["airdrop_start_time"]=$current_date' >"$TMP_GENESIS" && mv "$TMP_GENESIS" "$GENESIS"
-
-# Set claims records for validator account
-amount_to_claim=10000
-claims_key=${KEYS[0]}
-node_address=$(evmosd keys show $claims_key --keyring-backend $KEYRING --home "$HOMEDIR" | grep "address" | cut -c12-)
-cat "$GENESIS" | jq -r --arg node_address "$node_address" --arg amount_to_claim "$amount_to_claim" '.app_state["claims"]["claims_records"]=[{"initial_claimable_amount":$amount_to_claim, "actions_completed":[false, false, false, false],"address":$node_address}]' >"$TMP_GENESIS" && mv "$TMP_GENESIS" "$GENESIS"
-
-# Set claims decay
-cat "$GENESIS" | jq '.app_state["claims"]["params"]["duration_of_decay"]="1000000s"' >"$TMP_GENESIS" && mv "$TMP_GENESIS" "$GENESIS"
-cat "$GENESIS" | jq '.app_state["claims"]["params"]["duration_until_decay"]="100000s"' >"$TMP_GENESIS" && mv "$TMP_GENESIS" "$GENESIS"
-
-# Claim module account:
-# 0xA61808Fe40fEb8B3433778BBC2ecECCAA47c8c47 || evmos15cvq3ljql6utxseh0zau9m8ve2j8erz89m5wkz
-cat "$GENESIS" | jq -r --arg amount_to_claim "$amount_to_claim" '.app_state["bank"]["balances"] += [{"address":"evmos15cvq3ljql6utxseh0zau9m8ve2j8erz89m5wkz","coins":[{"denom":"aevmos", "amount":$amount_to_claim}]}]' >"$TMP_GENESIS" && mv "$TMP_GENESIS" "$GENESIS"
-
-# disable produce empty block
-if [[ "$OSTYPE" == "darwin"* ]]; then
-	sed -i '' 's/create_empty_blocks = true/create_empty_blocks = false/g' "$CONFIG"
+if [ -d "$HOMEDIR" ]; then
+	echo "An existing folder at '$HOMEDIR' was found. Enter Y to delete this folder and restart a new blockchain with new keys. Enter anything else to start the existing blockchain."
```

Comment:
nit, Its easier to read if the call to action starts on a new line and if its asked as a question. Currently on my device I get some other errors before too, so it might be nice to add an empty line before too. 

## Current 

```
➜ ./local_node.sh
go install -tags "netgo ledger" -ldflags '-X github.com/cosmos/cosmos-sdk/version.Name=evmos -X github.com/cosmos/cosmos-sdk/version.AppName=evmosd -X github.com/cosmos/cosmos-sdk/version.Version=29c03b5f -X github.com/cosmos/cosmos-sdk/version.Commit=29c03b5fc1f99f8b166b9c2889e15e3f2de4446a -X "github.com/cosmos/cosmos-sdk/version.BuildTags=netgo,ledger" -X github.com/tendermint/tendermint/version.TMCoreSemVer=v0.34.24 -w -s' -trimpath  ./...
# github.com/keybase/go-keychain
cgo-gcc-prolog:203:11: warning: 'SecTrustedApplicationCreateFromPath' is deprecated: first deprecated in macOS 10.15 - No longer supported [-Wdeprecated-declarations]
/Library/Developer/CommandLineTools/SDKs/MacOSX10.15.sdk/System/Library/Frameworks/Security.framework/Headers/SecTrustedApplication.h:59:10: note: 'SecTrustedApplicationCreateFromPath' has been explicitly marked deprecated here
An existing folder at '/Users/danielburckhardt/.tmp-evmosd' was found. Enter Y to delete this folder and restart a new blockchain with new keys. Enter anything else to start the existing blockchain.
```

## Suggestion

```
➜ ./local_node.sh
go install -tags "netgo ledger" -ldflags '-X github.com/cosmos/cosmos-sdk/version.Name=evmos -X github.com/cosmos/cosmos-sdk/version.AppName=evmosd -X github.com/cosmos/cosmos-sdk/version.Version=29c03b5f -X github.com/cosmos/cosmos-sdk/version.Commit=29c03b5fc1f99f8b166b9c2889e15e3f2de4446a -X "github.com/cosmos/cosmos-sdk/version.BuildTags=netgo,ledger" -X github.com/tendermint/tendermint/version.TMCoreSemVer=v0.34.24 -w -s' -trimpath  ./...
# github.com/keybase/go-keychain
cgo-gcc-prolog:203:11: warning: 'SecTrustedApplicationCreateFromPath' is deprecated: first deprecated in macOS 10.15 - No longer supported [-Wdeprecated-declarations]
/Library/Developer/CommandLineTools/SDKs/MacOSX10.15.sdk/System/Library/Frameworks/Security.framework/Headers/SecTrustedApplication.h:59:10: note: 'SecTrustedApplicationCreateFromPath' has been explicitly marked deprecated here

An existing folder at '/Users/danielburckhardt/.tmp-evmosd' was found. You can delete this folder and start a new local node from genesis. Else the existing node is resumed. 
Override exisiting node and restart a new local node? [y/N]